<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupBaseDialog</class>
 <widget class="QDialog" name="SetupBaseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>7</y>
     <width>751</width>
     <height>31</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>741</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="rdoExtra">
       <property name="text">
        <string>科目期初余额</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rdoProfit">
       <property name="text">
        <string>利润表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rdoAsset">
       <property name="text">
        <string>资产负债表</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>900</width>
     <height>560</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>900</width>
     <height>600</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="pageExtra">
    <property name="maximumSize">
     <size>
      <width>900</width>
      <height>550</height>
     </size>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>871</width>
       <height>511</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>900</width>
       <height>600</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>资产类</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>841</width>
         <height>461</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gl1"/>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>负债类</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>841</width>
         <height>341</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gl2"/>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>共同类</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>831</width>
         <height>141</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gl3"/>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>所有者权益类</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>831</width>
         <height>271</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gl4"/>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>成本类</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>841</width>
         <height>141</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gl5"/>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>损益类</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>831</width>
         <height>441</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gl6"/>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="btnExtraSave">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>520</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>520</y>
       <width>147</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Andale Mono</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>设置时间基点：</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="datExtra">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>520</y>
       <width>139</width>
       <height>27</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>yyyy-M</string>
     </property>
     <property name="calendarPopup">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEnd">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>520</y>
       <width>93</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>完成</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblSetp">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>516</y>
       <width>62</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Andale Mono</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>4/4</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pageProfit">
    <widget class="QTableView" name="tabProfit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>481</width>
       <height>531</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>511</x>
       <y>37</y>
       <width>261</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Andale Mono</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>报表数据年月：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDateEdit" name="datProfit">
        <property name="displayFormat">
         <string>yyyy-M</string>
        </property>
        <property name="calendarPopup">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="chkProfit">
        <property name="font">
         <font>
          <family>Andale Mono</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>年度余额</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnProfitSave">
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="pageAsset">
    <widget class="QTableView" name="tabAsset">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>481</width>
       <height>541</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>511</x>
       <y>33</y>
       <width>271</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Andale Mono</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>报表数据年月：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDateEdit" name="datAsset">
        <property name="displayFormat">
         <string>yyyy-M</string>
        </property>
        <property name="calendarPopup">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="chkAsset">
        <property name="font">
         <font>
          <family>Andale Mono</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>年度余额</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnAssetSave">
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnExtraSave</sender>
   <signal>clicked()</signal>
   <receiver>SetupBaseDialog</receiver>
   <slot>saveExtra()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>586</x>
     <y>571</y>
    </hint>
    <hint type="destinationlabel">
     <x>703</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdoExtra</sender>
   <signal>clicked()</signal>
   <receiver>SetupBaseDialog</receiver>
   <slot>switchPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>12</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdoProfit</sender>
   <signal>clicked()</signal>
   <receiver>SetupBaseDialog</receiver>
   <slot>switchPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>9</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdoAsset</sender>
   <signal>clicked()</signal>
   <receiver>SetupBaseDialog</receiver>
   <slot>switchPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>566</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>13</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnProfitSave</sender>
   <signal>clicked()</signal>
   <receiver>SetupBaseDialog</receiver>
   <slot>saveProfit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>790</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>875</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAssetSave</sender>
   <signal>clicked()</signal>
   <receiver>SetupBaseDialog</receiver>
   <slot>saveAsset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>781</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>879</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkProfit</sender>
   <signal>toggled(bool)</signal>
   <receiver>SetupBaseDialog</receiver>
   <slot>checkBoxToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>879</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkAsset</sender>
   <signal>toggled(bool)</signal>
   <receiver>SetupBaseDialog</receiver>
   <slot>checkBoxToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>585</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>888</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnEnd</sender>
   <signal>clicked()</signal>
   <receiver>SetupBaseDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>728</x>
     <y>573</y>
    </hint>
    <hint type="destinationlabel">
     <x>884</x>
     <y>562</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>datExtra</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>SetupBaseDialog</receiver>
   <slot>readExtraData(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>579</y>
    </hint>
    <hint type="destinationlabel">
     <x>12</x>
     <y>463</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>switchPage()</slot>
  <slot>saveExtra()</slot>
  <slot>saveProfit()</slot>
  <slot>saveAsset()</slot>
  <slot>checkBoxToggled(bool)</slot>
  <slot>readExtraData(QDate)</slot>
 </slots>
</ui>
